src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTagger.cs

```diff
@@ -28,7 +29,7 @@ internal sealed class InlineHintsTagger : ITagger<IntraTextAdornmentTag>, IDispo
         /// <summary>
         /// stores the parameter hint tags in a global location 
         /// </summary>
-        private readonly List<ITagSpan<IntraTextAdornmentTag>> _cache;
+        private readonly List<(IMappingTagSpan<InlineHintDataTag> mappingTagSpan, ITagSpan<IntraTextAdornmentTag>? tagSpan)> _cache;
```

Comment:
```suggestion
        private readonly List<(IMappingTagSpan<InlineHintDataTag> mappingTagSpan, ITagSpan<IntraTextAdornmentTag>? tagSpan)> _cache = new();
```

Reply:
This lets you remove the line in the constructor and keeps it easy to change in the future if needed

Reply:
that's a better pattern to have, changed.